build.gradle

```diff
@@ -2,7 +2,7 @@ ext {
     coroutinesVersion = '1.5.2'
     wordPressUtilsVersion = '2.3.0'
     wordPressLoginVersion = '0.0.8'
-    gutenbergMobileVersion = 'v1.71.0'
+    gutenbergMobileVersion = 'v1.71.1'
```

Comment:
`trunk` is still in `1.71.0` (i.e. no new version with a different version and conflicting number of Gutenberg has been made in trunk yet) so it makes sense to include this bump when merging this PR to `trunk` 👍 

We might want to close https://github.com/wordpress-mobile/WordPress-Android/pull/15948 (which was auto-generated by the Gutenberg PR bot but ends up duplicating the above change from this PR) once the current PR — which takes precedence — gets merged, btw (cc @fluiddot ?)

Reply:
Sure, we can safely close https://github.com/wordpress-mobile/WordPress-Android/pull/15948 now that this PR got merged 👍. On the other hand, I wasn't aware that we were automatically generating that kind of PRs to bump the Gutenberg Mobile version 🤩 .

Reply:
I think that comes from p9ugOq-2AI-p2 ? Which is also why I let you close the PR instead of doing it myself silently, so that you see this and might want to see if there's something worth improving in the automation for those kind of use cases or not 🤷

Reply:
PS: tbh given this is quite dependent on when those PRs happen, and the sync with us doing betas vs you landing fixes in trunk with different versions than the ones in release/* depending on what happens first… not sure it's worth trying to make the automation smarter (might end up to complex to handle all cases and timings); but it's just something to be aware of at least.

Reply:
> I think that comes from p9ugOq-2AI-p2 ? Which is also why I let you close the PR instead of doing it myself silently 😇, so that you see this and might want to see if there's something worth improving in the automation for those kind of use cases or not 🤷

We currently have the option to automatically create PRs in GB-mobile from the Gutenberg repo. I know there's an ongoing effort for doing something similar in WordPress repositories from GB-mobile PRs, but I didn't know that it was already available 😅 .

Btw, I've just seen that the PR is already closed ✅ . Thanks @AliSoftware for raising this topic as I wasn't aware 🙇 .